ui/widget.js

```diff
@@ -390,6 +408,55 @@ $.Widget.prototype = {
 		return this._setOptions({ disabled: true });
 	},
 
+	_classes: function( element, keys, extra, add, object ) {
```

Comment:
Argument names should make sense to other readers, and `object` has no meaning for me.